projects/cashmere/src/lib/button-toggle/button-toggle-group.component.ts

```diff
@@ -0,0 +1,173 @@
+
+import { AfterContentInit, Component, ContentChildren, EventEmitter, HostBinding, Input, OnDestroy, Output, ViewEncapsulation } from '@angular/core';
+import { QueryList } from '@angular/core';
+import { Subject } from 'rxjs';
+import { takeUntil } from 'rxjs/operators';
+import { ButtonToggleChangeEvent } from './button-toggle-change-event';
+import { parseBooleanAttribute } from '../util';
+import { ButtonToggleComponent } from './button-toggle.component';
+
+const supportedStyles = ['primary', 'primary-alt', 'destructive', 'neutral', 'secondary'];
+const supportedColors = ['blue', 'green', 'purple', 'red', 'orange', 'ruby-red', 'deep-red', 'red-orange', 'magenta', 'pink', 'light-pink', 'azure', 'teal', 'dark-green', 'brown', 'purple-gray', 'yellow', 'yellow-orange', 'tan'];
+const supportedSizes = ['sm', 'md', 'lg'];
+
+export function validateStyleInput(style: string) {
+    if (supportedStyles.indexOf(style) < 0 && supportedColors.indexOf(style) < 0) {
```

Comment:
Should this be an OR here so that if one of them isn't supported, the whole thing fails?

Reply:
Oh, looks like this might be more like "Check if `primary` is valid" and it'll check both arrays?

Reply:
Yeah - this stuff is actually stolen from other components and deposited untouched.

Reply:
Might be nice to stick this stuff in a shared file somewhere for both components to use.

Reply:
@corykon - I'm all for avoiding redundant code.  It looks like these validation functions are already being shared by split-button and load-more-pagination.  I'm going to push one more commit that has the button-toggle pull from the shared location as well.